### PR TITLE
Add node version check to system health check

### LINKED ISSUE #1085: More complete system check
Check minimum node version.
Keep the system check visible, but collapsed if all checks pass.

Add a green checkmark if all tests pass

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
Closes #1085

### CODE REVIEW

**Review by jchenche — APPROVED** [review]

File: taqueria-vscode-extension/package.json [hunk 1]
```diff
@@ -106,64 +106,90 @@
...
+			},
+			{
+				"view": "taqueria-system-check",
+				"when": "!@taqueria-state/is-taq-cli-reachable",
+				"contents": "❌ Taq CLI: Unable to find the Taqueria CLI.\nThe Taqueria CLI is required to use this extension. Click [here](https://taqueria.io/docs/getting-started/installation/) for installation instructions, then click the \"Check Again\" button.\nThe Taqueria CLI must also be in your PATH. If you change your PATH, you might need to restart Visual Studio Code to register this change."
+			},
+			{
+				"view": "taqueria-system-check",
+				"when": "@taqueria-state/is-node-installed && @taqueria-state/node-version-meets-requirements",
+				"contents": "✅ NodeJs: Installed"
```
Comment (hu3man) [inline taqueria-vscode-extension/package.json:178]: `NodeJs` ideally should be `Node.js`

If a period in a line is going to cause a headache, the former is fine though

File: taqueria-vscode-extension/src/lib/helpers.ts [hunk 2]
```diff
@@ -16,12 +17,15 @@ import { EnvironmentsDataProvider } from './gui/EnvironmentsDataProvider';
...
 import { SandboxesDataProvider, SandboxTreeItem } from './gui/SandboxesDataProvider';
 import { ScaffoldsDataProvider, ScaffoldTreeItem } from './gui/ScaffoldsDataProvider';
+import { SystemCheckDataProvider, SystemCheckTreeItem } from './gui/SystemCheckDataProvider';
 import { TestDataProvider, TestTreeItem } from './gui/TestDataProvider';
 import * as Util from './pure';
 import { TaqVsxError } from './TaqVsxError';
 
 export const COMMAND_PREFIX = 'taqueria.';
 
+const minNodeVersion = '16.10.0';
```
Comment (hu3man) [inline taqueria-vscode-extension/src/lib/helpers.ts:27]: Have we confirmed that this is the case? I was under the impression we needed `16.13.1`

Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/src/lib/helpers.ts]: Done, thanks

File: taqueria-vscode-extension/package.json (hunk 1 above)
Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/package.json]: Done. Thanks.

**Review by hu3man — APPROVED** [review]

**Pull request merged**